src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -185,26 +197,40 @@ def _get_variance(self, timestep, prev_timestep):
 
         return variance
 
-    def set_timesteps(self, num_inference_steps: int, device: Union[str, torch.device] = None):
+    def set_timesteps(
+        self,
+        num_inference_steps: int,
+        device: Union[str, torch.device] = None,
+        substeps_mode: str = "linear",
```

Comment:
As mentioned above, I'm not sure we want to add this to the native DDIM scheduler